components/gitpod-db/src/typeorm/migration/1642497869312-ProjectInfo.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class ProjectInfo1642497869312 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query("CREATE TABLE IF NOT EXISTS `d_b_project_info` ( `projectId` char(36) NOT NULL, `overview` longtext NOT NULL, `deleted` tinyint(4) NOT NULL DEFAULT '0', `_lastModified` timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6), PRIMARY KEY (`projectId`), KEY `ind_dbsync` (`_lastModified`)) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;");
```

Comment:
Note: I'm using `longtext` for `overview` instead of just `text` because the (enormous) Project Overview of the GitLab repository (my stress test) wouldn't fit into a simple `text`.

We could also look into trimming unnecessary information from Project Overviews, but I'd leave this as a potential future optimization.

Reply:
We should look into this issue. I assume it's because of commit message length.